### PR TITLE
Add test suite

### PR DESCRIPTION
**Pull request by aneeshusa**
Currently checks for executable shebangs and trailing whitespace.
Adds a dispatch script to run on Travis.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/295)

### CODE REVIEW

**Comment by aneeshusa** [comment]
Rebased on top of the latest master, the homu update also fixed a whitespace bug.

File: .travis.yml [hunk 1]
```diff
@@ -20,23 +20,16 @@ matrix:
       os: linux
       sudo: required
       dist: trusty
+    - env: SALT_NODE_ID=test # Not a Salt node, runs test suite instead
+      os: linux
+      sudo: required
+      dist: trusy
```
Comment (Wafflespeanut) [inline .travis.yml]: um, trusty? :)

Comment (aneeshusa) [inline .travis.yml]: Oops, thanks!

**Comment by larsbergstrom** [comment]
Reviewed 9 of 9 files at r1.
Review status: 7 of 12 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/servo/saltfs/295)*

**Comment by aneeshusa** [comment]
I updated the Salt install instructions and mentioned the test script on the wiki.

**Pull request merged**